cricPredict/Contest/views.py

```diff
@@ -17,6 +17,7 @@
 
 class LeagueAPIView(APIView):
     permission_classes = [IsAuthenticated]
+    authentication_classes = [JWTAuthentication]
```

Comment:
I thought we agreed on configuring the authentication in a way that we don't need to specify `authentication_classes` on each view

Reply:
I forgot to remove these from views